src/components/token-family/TokenFamilyHeader.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useMemo, useRef } from 'react';
+import { Animated, Easing, StyleSheet, View } from 'react-native';
+import FastImage from 'react-native-fast-image';
+import CaretImageSource from '../../assets/family-dropdown-arrow.png';
+import { useLatestCallback } from '../../hooks';
+import { ButtonPressAnimation } from '../animations';
+import TokenFamilyHeaderIcon from './TokenFamilyHeaderIcon';
+import { Text } from '@rainbow-me/design-system';
+import { ImgixImage } from '@rainbow-me/images';
+import { ThemeContextProps } from '@rainbow-me/theme';
+
+export const TokenFamilyHeaderAnimationDuration = 200;
+export const TokenFamilyHeaderHeight = 50;
+
+const sx = StyleSheet.create({
+  amountContainer: {
+    marginRight: 10,
+  },
+  center: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    justifyContent: 'center',
+  },
+  chevron: {
+    height: 18,
+    width: 8,
+  },
+  content: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    height: TokenFamilyHeaderHeight,
+    justifyContent: 'space-between',
+    padding: 19,
+    paddingRight: 14,
+    width: '100%',
+  },
+  title: {
+    flex: 1,
+    paddingRight: 9,
+  },
+});
+
+const TokenFamilyHeader = ({
+  childrenAmount,
+  emoji,
+  familyImage,
+  isOpen,
+  onPress,
+  testID,
+  title,
+  theme,
+}: {
+  childrenAmount?: number;
+  emoji?: string;
+  familyImage?: string;
+  isOpen?: boolean;
+  onPress?: () => void;
+  testID?: string;
+  title: string;
+  theme: ThemeContextProps;
+}) => {
```

Comment:
Could you extract interface to be separate and not inline? IMO it is way cleaner. Let me know what you think